src/test/operators/export_test.cpp

```diff
@@ -123,37 +123,34 @@ TEST_F(OperatorsExportTest, NonsensePath) {
   table->append({1, "hello", 3.5f});
   auto table_wrapper = std::make_shared<TableWrapper>(std::move(table));
   table_wrapper->execute();
-  auto exporter = std::make_shared<opossum::Export>(table_wrapper, "this/path/does/not/exist");
+  auto exporter = std::make_shared<opossum::Export>(table_wrapper, "this/path/does/not/exist.tbl");
   EXPECT_THROW(exporter->execute(), std::exception);
 }
 
-TEST_F(OperatorsExportTest, EmptyPath) {
+TEST_F(OperatorsExportTest, EmptyPathWithoutFileType) {
   table->append({1, "hello", 3.5f});
   auto table_wrapper = std::make_shared<TableWrapper>(std::move(table));
   table_wrapper->execute();
-  auto exporter = std::make_shared<opossum::Export>(table_wrapper, "");
-  EXPECT_THROW(exporter->execute(), std::exception);
+
+  EXPECT_THROW(std::make_shared<opossum::Export>(table_wrapper, ""), std::exception);
 }
 
-TEST_F(OperatorsExportTest, UnknownFileExtension) {
+TEST_F(OperatorsExportTest, EmptyPathWithFileType) {
   table->append({1, "hello", 3.5f});
   auto table_wrapper = std::make_shared<TableWrapper>(std::move(table));
   table_wrapper->execute();
-  auto exporter = std::make_shared<opossum::Export>(table_wrapper, "not_existing_file.mp3");
+
+  auto exporter = std::make_shared<opossum::Export>(table_wrapper, "", FileType::Csv);
+
   EXPECT_THROW(exporter->execute(), std::exception);
 }
 
-TEST_F(OperatorsExportTest, ReturnsInput) {
-  auto table = load_table("resources/test_data/tbl/float.tbl");
+TEST_F(OperatorsExportTest, UnknownFileExtension) {
+  table->append({1, "hello", 3.5f});
   auto table_wrapper = std::make_shared<TableWrapper>(std::move(table));
   table_wrapper->execute();
-  auto exporter = std::make_shared<opossum::Export>(table_wrapper, test_filename + ".bin");
-  exporter->execute();
-
-  // need to load table again since it moved
-  table = load_table("resources/test_data/tbl/float.tbl");
 
-  EXPECT_TABLE_EQ_ORDERED(exporter->get_output(), table);
+  EXPECT_THROW(std::make_shared<opossum::Export>(table_wrapper, "not_existing_file.mp3"), std::exception);

```

Comment:
Not specific to this line. I just chose a random place to put this comment. Should we test that COPY FROM/IMPORT and COPY TO/EXPORT are correctly translated with SQLTranslatorTests?

Reply:
I will add this

Reply:
This could also be an operator test, couldn't it?

Reply:
I'm not sure since we want to check if the input operator of the ExportOperator is a ValidateOperator in the LQP.  So this is kind of out of scope of the operator test. We check if a ValidateOperator is part of the LQP when the SQL query is translated with UseMvcc::True, but I thought more of an integration test (so if everything works when running the server).

Reply:
Ah, I see, I misread your previous post. Checking the correct input for the ExportOperator could be part of the SQLTranslatorTest?

Reply:
Also, we could add a note at the end of sqlite_testrunnter_queries.sql that we cannot test copy there.

Reply:
This is what I added recently, we ensure to have a ValidateNode as input of the ExportNode when UseMvcc::Yes.

Reply:
Discussed offline.